### PR TITLE
Add tips in UG

### LINKED ISSUE #203: Add tips to every command in UG

### PR DESCRIPTION
**Pull request by tayxuenye**
Added tips for all commands except Exit as I find it a bit pointless.

Please check carefully as some commands were not implemented by me.

Resolves #203

### CODE REVIEW

**Comment by Tkaixiang** [comment]
Fixes #203

File: docs/UserGuide.md [hunk 1]
```diff
@@ -335,6 +379,10 @@ Format: `listing delete INDEX LISTING_INDEX`
 * The `LISTING_INDEX` refers to the index number shown in the property listing displayed by the `show` command
 * The `INDEX`/`LISTING_INDEX` **must be a positive integer** 1, 2, 3, …​
 
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:** 
+Use `show` command to see the updated listings.
+</div>
+
```
Comment (Tkaixiang) [inline docs/UserGuide.md:385]: I will be updating this as a function in a different PR

**Review by Tkaixiang — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md [hunk 2]
```diff
@@ -174,6 +182,10 @@ Shows a list of all persons in the PROperty.
 
 Format: `list`
 
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:** 
+Use `list` after using commands which edit the data to see the updates. 
+</div>
+
```
[the remark this answers is not in the document]
Comment (tayxuenye) [reply docs/UserGuide.md:188]: Use `list` for a quick overview of all your contacts.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -212,6 +232,10 @@ Shows commands in a help menu for quick reference during use of PROperty.
 
 Format: `help`
 
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:** 
+Use `help` if you forget the command format. 
+</div>
+
```
[the remark this answers is not in the document]
Comment (tayxuenye) [reply docs/UserGuide.md:238]: Use `help` if you need help with any command usage or if you forget what commands there are.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -292,6 +328,10 @@ Format: `remark INDEX r/[REMARKS]`
 * The index **must be a positive integer** 1, 2, 3, …​
 * **Note:** `[REMARKS]` will **delete the remark if left blank**
 
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:** 
+Remarks are helpful to add details about a contact in a non-fixated format.
```
[the remark this answers is not in the document]
Comment (tayxuenye) [reply docs/UserGuide.md:332]: Remarks are useful for adding free-form details to a contact.

File: docs/UserGuide.md [hunk 5]
```diff
@@ -262,6 +290,10 @@ Format: `findtag TAG [MORE_TAGS]`
 * Persons with at least one matching tag will be returned (i.e., an `OR` search). 
 * List of possible tags you can search for are found in the [Tag Table](#tag-table)
 
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:** 
+Tags make it easy for you to find your target audience. 
+</div>
+
```
[the remark this answers is not in the document]
Comment (tayxuenye) [reply docs/UserGuide.md]: Tags make it easy for you to categorise your contacts into different groups.

**Comment by tayxuenye** [comment]
Please review again. Changes suggested have been addressed. Thanks!

**Review by Tkaixiang — APPROVED** [review]
pre-approved

**Pull request merged**